### DISCUSSION TITLE
Where can i find Input Shapping menu in Screen?

### DISCUSSION THREAD

**Discussion by gsuresh2u** [mriscoc/Ender3V2S1#740] [Q&A]
Hi, i am using SKR MINI E3 V3 on Ender 3 V2
updated screen firmware and board firmware
but i cant find any modificaion menu for Input Shappig and altering Linear Advance K value
attaching my configs, what i am doing wrong, please help
[configs.zip](https://github.com/mriscoc/Ender3V2S1/files/10751678/configs.zip)

**Comment by mriscoc** [comment]
Check in https://github.com/mriscoc/Special_Configurations

Advance K, and Input Shaping settings are in Control / Motion menu.

**Reply by gsuresh2u** [reply]
Found Advance K in Control --> Motion menu
but not found Input Shapping, Please help

**Reply by mriscoc** [reply]
Only firmwares with IS enabled show IS settings
check https://github.com/mriscoc/Special_Configurations/releases/tag/InputShaping

**Comment by gsuresh2u** [comment]
Yes, input Shaping is enabled

**Reply by gsuresh2u** [reply]
Sir, enabled Input Shapping, still not showing in Config --> Motion menu
![Untitled](https://user-images.githubusercontent.com/39871361/219586257-f6fca913-9c2a-4502-b412-e765890761c7.jpg)

attaching my config files, please check
[configs.zip](https://github.com/mriscoc/Ender3V2S1/files/10764594/configs.zip)

**Comment by gsuresh2u** [comment]
Sir, please check my problem

**Reply by fuzelet** [reply]
You need to enable "Shaping_Menu" to see it once you enable Input_Shaping.

**Comment by mriscoc** [comment]
Use the Configurator to enable Input Shaping (IS) in the features list:

![](https://github.com/mriscoc/Special_Configurations/raw/main/images/Configurator.png)

**Comment by gsuresh2u** [comment]
OK, got it

**Reply by ghost** [reply]
do you have Ender3v2 - SKRME3V3 - DWIN - BLT x UBL - T1 - IS - LA - MPC - compiled?
